pop-api/README.md

```diff
@@ -42,32 +53,55 @@ Certain types are shared between the ink! library portion of the Pop API and the
 
 When we use the Pop API in our smart contract like so:
 ```rust
-use pop_api::nfts::*;
-mint(collection_id, item_id, receiver)?;
+use pop_api::fungibles::{self as api};
+// -- snip --
+#[ink(message)]
+pub fn transfer(&mut self, token: TokenId, to: AccountId, value: Balance) -> Result<()> {
+    // Use of Pop API to call into the runtime to transfer some fungible assets.
+    api::transfer(token, to, value)
+}
 ```
 
-This will call the Pop API `mint` function in the [./src/v0/nfts.rs](./src/v0/nfts.rs) file, which is a wrapper to `dispatch` a `Runtime::Call` to the NFTs pallet's mint function. This is how most of the Pop API is built. This abstraction allows for creating a developer-friendly API over runtime level features such as calling pallets, reading state, and cross-chain interactions. All Pop API functionality can be found in [./src/v0/](./src/v0/) which is the current version of the Pop API.
+This will call the Pop API `transfer` function in the [./src/v0/fungibles/mod.rs](./src/v0/fungibles/mod.rs) file, which is a wrapper to `dispatch` a `Runtime::Call` to the assets pallet's transfer function. This is how most of the Pop API is built. This abstraction allows for creating a developer-friendly API over runtime level features such as calling pallets, reading state, and cross-chain interactions. All Pop API versions can be found in [pop-api/src/](./src/).
 
 
 ## Dispatching to the runtime ([./src/lib.rs](./src/lib.rs))
 
-### `PopApi` 
-The `PopApi` trait is an ink! chain extension trait with three functions:
-- `dispatch()`
-- `read_state()`
-- `send_xcm()`
+### `PopApi`
```

Comment:
We are not using this name anymore. The equivalent in the current implementation would be `ChainExtensionMethodApi`. Thought that keeping the reference as `PopApi` made the document easier to read.

But if this can be misleading we should correct it.